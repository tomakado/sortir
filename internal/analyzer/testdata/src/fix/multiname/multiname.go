package multiname

var (
	z, x, y int
	c, a, b string // want "variable/constant declarations are not sorted"
)

const (
	Z, X, Y = 3, 1, 2
	C, A, B = "c", "a", "b" // want "variable/constant declarations are not sorted"
)