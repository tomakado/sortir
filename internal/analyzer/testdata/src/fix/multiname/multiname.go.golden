package multiname

var (
	a, b, c string // want "variable/constant declarations are not sorted"
	x, y, z int    // want "variable/constant declarations are not sorted"
)

const (
	A, B, C = "c", "a", "b" // want "variable/constant declarations are not sorted"
	X, Y, Z = 3, 1, 2       // want "variable/constant declarations are not sorted"
)